src/ParachainRegistry.sol

```diff
@@ -38,14 +40,22 @@ contract ParachainRegistry is IRegistry {
     /// @dev Register parachain, along with index of Tellor pallet within corresponding runtime.
     /// @param _paraId uint32 The parachain identifier.
     /// @param _palletInstance uint8 The index of the Tellor pallet within the parachain's runtime.
-    function register(uint32 _paraId, uint8 _palletInstance) external {
+    /// @param weightToFee uint256 The constant multiplier(fee per weight) used to convert weight to fee
+    /// @param feeLocation XcmTransactorV2.Multilocation The location of the currency type of consumer chain.
+    function register(
+        uint32 _paraId,
+        uint8 _palletInstance,
+        uint256 weightToFee,
```

Comment:
These two parameters should have leading underscore in name as with others please. Dont forget to update the function signature comments accordingly.